src/blocks/testimonials/save.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies.
+ */
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+
+export default function save( { attributes } ) {
+	const {
+		columns,
+		gutter,
+		styleName,
+	} = attributes;
+
+	const saveBlockProps = useBlockProps.save();
+
+	// Remove classes related to colors as we don't want to apply them on the main block.
+	const sanitizedClasses = ( receivedClasses ) => {
+		const classesArray = receivedClasses.split( ' ' );
+		return classesArray.filter( ( classe ) => ! classe.match( new RegExp( 'has-', 'g' ) ) || classe.match( new RegExp( 'font-size', 'g' ) ) );
+	};
+
+	const classes = classnames( sanitizedClasses( saveBlockProps.className ), {
+		'has-columns': columns > 1,
+		'has-responsive-columns': columns > 1,
+		[ `has-${ columns }-columns` ]: columns > 1,
+		[ `has-${ gutter }-gutter` ]: gutter,
```

Comment:
the `has-gutter-gutter` class logic comes from our Gutter Controls component. 
https://github.com/godaddy-wordpress/coblocks/blob/de4b025d4691febe34e37d78b63a485363ce3757/src/components/gutter-control/gutter-wrapper.js#L34

We have a number of examples through the CoBlocks codebase where we use this component and controls.